### ISSUE TITLE
Improve installation instructions, maybe add more details about unofficial packages

### ISSUE THREAD

**Issue by simskij** [grafana/k6-docs#52]
Like asdf, aur, appimage, snap among others.

Should also include a disclaimer about these methods not being official

**Comment by simskij** [comment]
Same as mentioned in #51. Closing that one as this contains (somewhat) more details. Thanks @MStoykov ! 🙏

**Comment by na--** [comment]
https://repology.org/project/k6/versions

We can mention some of these packages - they're unofficial for us, in that we don't build them ourselves, but they may be official to some of the distributions (i.e. they are officially packaged in their main distribution channels)... So, I think we should distinguish between at least 3 tiers of trust:
1. Managed by us
2. First-class packages in a distribution 
3. Just a random package built by a random person on the internet

The distinction between 2 and 3 for me is whether someone from the distribution has to review the package before it can be updated, or if a single random person on the internet can release a new bogus k6 version with malware via that channel... :smile: Homebrew and nixpkgs probably fall in the second category, while AUR is in the third, I think...

**Comment by MStoykov** [comment]
- the [asdf is one](https://kandi.openweaver.com/shell/grimoh/asdf-k6) year old
- aur is [up to date](https://aur.archlinux.org/packages/k6)
- can't find an appimage
- [snap](https://snapcraft.io/k6) is 3 years old.

Given that I kind of feel like there is barely any reason to make this page. 

I would argue Arch Linux users are aware AUR exists and know how to check it and have only them seems ... strange

So I am for closing this and *not* do anything. So for the purposes on not waiting on this I *am* closing this issue and if someone disagrees they can reopen it.

**Comment by MStoykov** [comment]
The [chocolatey one](https://community.chocolatey.org/packages/k6) seems up to date and has had a [PR](https://github.com/grafana/k6/pull/1027) to be added to the README ~forever~ three years.

**Comment by na--** [comment]
I somewhat disagree here. Maybe we don't need a separate page, but I don't think the [current page with installation instructions](https://k6.io/docs/getting-started/installation/) is very good. So I'll reopen this with a slightly changed title.

**Comment by MStoykov** [comment]
I would prefer if you open a new issue and propose the changes you want or at least describe what you don't like from the current one.

Now this issue has a title and first comment that don't go all that well together

**Comment by MattDodsonEnglish** [comment]
The installation page has instructions for the choco package install now.  Same one documented in the linked PR: https://github.com/grafana/k6/pull/1027/files

Closing this issue.
